requirements/tests.txt

```diff
@@ -5,3 +5,4 @@ mozprocess >= 0.22
 pytest==4.1.0
 pytest-cov==2.6.1
 wptserve >= 1.4.0
+pytest-mock >= 1.10.0
```

Comment:
Please keep the alphabetical order of inputs and make it strictly requiring this specific version.

Reply:
Should it be done for wptserve too?. Or is there any specific reason to keep it as >=.

Reply:
As best yes. Looks like that this slipped through.